nmigen_boards/zturn_lite.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.xilinx_7series import *
+from .dev import *
+
+__all__ = ["ZTurnLitePlatform"]
+
+
+class ZTurnLitePlatform(Xilinx7SeriesPlatform):
+    device = "xc7z010"
+    package = "clg400"
+    speed = "1"
+
+    resources = []
+
+    connectors = [
+        Connector(
+            "expansion", 0, 
+            "-   -  "
+            "B19 E17"
```

Comment:
This isn't correct. The concatenated string looks like `"... B19 E17A20 D18 ..."`